src/lib/Scenes/Map/GlobalMap.tsx

```diff
@@ -47,10 +60,16 @@ const ButtonAnimation = {
   },
 }
 
+enum DrawerPosition {
+  open = "open",
+  closed = "closed",
+  collapsed = "collapsed",
+  partiallyRevealed = "partiallyRevealed",
+}
```

Comment:
Shame that TypeScript can't infer the underly enum values à la Swift.

Reply:
Yup, that would be nice indeed